libsplinter/src/biome/credentials/store/diesel/mod.rs

```diff
@@ -14,7 +14,7 @@
 
 pub(in crate::biome) mod models;
```

Comment:
Could you add something to the commit message explaining that this so it can be used `update_keys_and_password`, or squash it into that commit?